### PR TITLE
use github actions instead

### PR DESCRIPTION
**Pull request by coco875**

### CODE REVIEW

**Comment by coco875** [comment]
image
what it look like once it's merged

**Comment by coco875** [comment]
fix non matching being on the second line

File: README.md [hunk 1]
```diff
@@ -39,7 +41,6 @@ Extract assets
 ```
 make assets -j
 ```
-```
```
Comment (MegaMech) [inline README.md:42]: You're missing ```

File: progress.py [hunk 2]
```diff
@@ -523,5 +524,46 @@ def list_detailed_progress_and_files(pct, prog, size, objlist, charseg):
...
+    badge_total_size = badge(left_text="Total size", right_text=str(total_code_size), right_color="blue")
+    with open("docs/html/total_size.svg", "w") as f:
+        f.write(badge_total_size)
+    badge_total_pct = badge(left_text="Total progress", right_text=str(round(totalPct, 2))+"%", right_color="green")
+    with open("docs/html/total_progress.svg", "w") as f:
+        f.write(badge_total_pct)
+    badge_game_pct = badge(left_text="Game progress", right_text=str(round(gamePct, 2))+"%", right_color="green")
+    with open("docs/html/game_progress.svg", "w") as f:
+        f.write(badge_game_pct)
+    badge_asm_funcs = badge(left_text="ASM functions", right_text=str(TotalGlobalAsmFunctions), right_color="blue")
```
Comment (MegaMech) [inline progress.py:543]: How do you feel about changing this to WIP functions?

File: README.md [hunk 3]
```diff
@@ -11,24 +11,26 @@ It supports and builds the following versions:
 | mk64.eu-1.0.z64   | EUR 1.0 | a729039453210b84f17019dda3f248d5888f7690 |
 | mk64.eu-final.z64 | EUR 1.1 | f6b5f519dd57ea59e9f013cc64816e9d273b2329 |
 
+status: [![Linux Compile test](https://github.com/n64decomp/mk64/actions/workflows/linux-compile.yml/badge.svg)](https://github.com/n64decomp/mk64/actions/workflows/linux-compile.yml)
```
Comment (MegaMech) [inline README.md:14]: Remove `status:` and rename `Linux Compile test` to `Linux compile`.

**Review by MegaMech — CHANGES_REQUESTED** [review]

**Review by MegaMech — APPROVED** [review]

**Pull request merged**